src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,55 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int SIZE = 10;
+    private Object[] keys;
+    private Object[] values;
```

Comment:
use parameters of class instead of just Object
```suggestion
    private K[] keys;
    private V[] values;
```